benchmarks/run-benchmark.sh

```diff
@@ -3,12 +3,47 @@
 # Runs a benchmark using drun. The script assumes drun is available on the caller's path.
 set -euo pipefail
 
+# Remove downloaded didc, drun.
+trap 'set +e && rm drun didc && set -e' EXIT SIGINT
+
 BENCH_NAME=$1
 FILE=$(mktemp)
 
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Add script directory to the PATH
+export PATH="$SCRIPT_DIR:$PATH"
```

Comment:
Do we need to add drun and didc to the path? I don't think we should add drun and didc to the path, as it might conflict with other versions of drun and didc that are already on the user's path.